### DISCUSSION TITLE
Option to turn off corner rounding on clip handles

### DISCUSSION THREAD

**Discussion by Jacqland** [audacity/audacity#2789] [Ideas & Feature Requests]
The problem: Currently, the tops of clips have rounded corners. This can make it difficult to see quickly/visually if multiple tracks are aligned because there's a little gap at the top of each one for the rounded corner.

The solution: In preferences or "view", allow users to select whether to turn clip rounding "on" or "off". This could also apply to the bottom of clips if people wanted (since they are currently unrounded).

Alternatives: I'm not sure. Maybe a way to permanently turn on "snap guides" or otherwise visually signal the time position of multiple clips? Or a way to turn off clip handles completely, though that would be annoying in a different way.

Additional context: I don't have any, but import any track into audacity and you will see what I mean with the rounding and how it's hard to see at-a-glance if things are lined up without checking every single track for a snap guide (which itself can introduce more issues).

**Comment by LWinterberg** [comment]
Unfortunately, just getting rid of the rounded corners won't massively aid your quest of aligning the clips:

<details>
<summary>screenshot example</summary>

![Screenshot from 2022-04-09 10-35-41](https://user-images.githubusercontent.com/87814144/162563868-65823ec2-0794-4eda-a89b-2a987adc47ef.png)

Which two of these clips are perfectly aligned?
</details>

However, it is planned to add a grid to align clips to beats and measures.

**Reply by Jacqland** [reply]
Counterpoint: While I can't tell at a glance if any of the tracks in your image are aligned, I can _clearly_ see the bottom two tracks aren't aligned, while this would be much more difficult to see with corner rounding. 

Here is an example with the same thing using both rounded and unrounded corners (four tracks, with the second misaligned). In the new version:
![audnew](https://user-images.githubusercontent.com/26366747/162568489-43ebf533-bef7-4107-ba91-f66e47c9b2ed.png)

It is much more difficult to see when compared to a version without rounding, specifically the bottom of the first track with the top of the second track, and the bottom of the second track with the top of the third track:
![audold](https://user-images.githubusercontent.com/26366747/162568496-4b816cac-aa70-44ef-b4bc-e7c99658dd29.png)

This may be partially because I am more used to seeing it and working with older versions of Audacity, so I am used to aligning the times with the top of the clip and this this "feature" of rounded corners is very difficult, compared to someone who doesn't have this workflow. With the rounding, I even find it difficult to know if I am aligning a single track correctly with the time I intend to, as the curve interrupts the line of the track with the time ticks along the top.

(I edited this post because originally I was not using the same zoom level on both images so it was a bit unfair. They are zoomed to the same level now.)
